RedacteurPortaal.Tests/Grains/ArchiveGrainTests.cs

```diff
@@ -41,10 +42,56 @@ public async Task CanAddArchiveCorrectly()
         Assert.Equal(toSaveArchive.MediaVideoItems, updatedArchive.MediaVideoItems);
         Assert.Equal(toSaveArchive.MediaPhotoItems, updatedArchive.MediaPhotoItems);
         Assert.Equal(toSaveArchive.Scripts, updatedArchive.Scripts);
+    }
 
+    [Fact]
+    public async Task CanGetASingularArchiveCorrectly()
+    {
+        var guid = Guid.NewGuid();
 
+        var toSaveArchive = new ArchiveModel(guid, "Title", "Label", new List<MediaPhotoItem> { }, new List<MediaVideoItem> { }, new List<MediaAudioItem> { }, new List<NewsItemModel> { }, DateTime.UtcNow, new List<string> { });
+        var archiveGrain = this._cluster.GrainFactory.GetGrain<IArchiveGrain>(guid);
+        await archiveGrain.CreateArchive(toSaveArchive);
 
+        var archiveInGrain = await this._cluster.GrainFactory.GetGrain<IArchiveGrain>(guid).Get();
 
+        Assert.NotNull(archiveInGrain);
     }
 
+    [Fact]
+    public async Task UpdateArchiveCorrectly()
+    {
+        var guid = Guid.NewGuid();
+
+        var toSaveArchive = new ArchiveModel(guid, "Title", "Label", new List<MediaPhotoItem> { }, new List<MediaVideoItem> { }, new List<MediaAudioItem> { }, new List<NewsItemModel> { }, DateTime.UtcNow, new List<string> { });
+        var archiveGrain = this._cluster.GrainFactory.GetGrain<IArchiveGrain>(guid);
+        await archiveGrain.CreateArchive(toSaveArchive);
+
+        var newerArchive = new ArchiveModel(guid, "Newer Title", "Newer Label", new List<MediaPhotoItem> { }, new List<MediaVideoItem> { }, new List<MediaAudioItem> { }, new List<NewsItemModel> { }, DateTime.UtcNow, new List<string> { });
+
+        await archiveGrain.Update(newerArchive);
+
+
+        var archiveInGrain = await this._cluster.GrainFactory.GetGrain<IArchiveGrain>(guid).Get();
+        Assert.NotEqual(toSaveArchive.Title, archiveInGrain.Title);
+        Assert.NotEqual(toSaveArchive.Label, archiveInGrain.Label);
```

Comment:
Zijn dit de enige properties in archive grain state? 
Je moet hier ook testen ov de nieuwe values gelijk zijn aan je update, want als create werkt, maar update alles naar null zet zou deze test ook passen

Reply:
De rest van de properties worden nu ook getest